proposals/tf_utf8_identifiers/README.md

```diff
@@ -0,0 +1,435 @@
+
+When early requirements for UTF-8 identifiers in Omniverse arose from German use cases, we initially advocated for a `displayName` approach that replaced umulauts with trailing e's and ß with "ss" in prim names.  However, once Japanese use cases came to the forefront as well, we reconsidered how lossy ASCII approximations could be.  While it is possible to Romanize Japanese kanji into hiragana and / or katakana with phonetic Latin-based text, it is not an automated process, and it's still difficult to navigate the stage using `GetPrimAtPath` calls.  That is, for the target workflows in AEC and manufacturing, the UTF-8 encoded namespace hierarchy really is the ground truth of the scene originating from source assets that travel the pipeline across DCCs such as Revit and 3ds Max.  It is cumbersome and disruptive to Romanize this ground truth in pipelines that are otherwise largely non-Romanized.  Even if the UI is showing the UTF-8 encoded `displayName` metadata, the user must still use Romanized names to navigate the scene hierarchy as described above.  We posit that navigating the scene hierarchy is a fundamental operation in USD that deserves deeper consideration for non-Latin based identifiers.  A `displayName` approach would either sacrifice the performance of natively querying prims by path, or incur potentially significant memory usage for large scene hierarchies to cache `displayName` lookups in place of native queries.
+
+Finally, other languages do not have standard Romanizations, and in general a Romanized character does not have a 1:1 correspondence with its source character.  For example, the Jyutping Romanization "sam1" maps to several unrelated Cantonese characters such as 心, 森, and 深.  This further fences off content creators the world over from leveraging and improving upon USD as the canonical mechanism for scene interchange across an ever-expanding ecosystem of tooling and pipelines in the virtual space.
+
+# An Overview of Identifiers in USD
+
+Identifier syntax is currently enforced in USD by several different mechanisms.  The primary mechanism is the inlined function `TfIsValidIdentifier`, which states that "an identifier is valid if it follows the C/Python identifier convention; that is, it must be at least one charaacter long, must start with a letter or underscore, and must contain only letters, underscores, and numerals".  However, there are several other code paths that validate different flavors of identifiers, including namespaced and variant identifiers, that use similar logic to achieve validation.  In particular, this means that any change to the logic governing core identifier validation in `TfIsValidIdentifier` necessitates change in several different similar validation functions across the `Sdf` module (for example, `SdfPath::IsValidNamespacedIdentifier`).
```

Comment:
Should charaacter be character?

Reply:
fixed

Reply:
This still seems to be there in the latest unless I'm missing an update.

Reply:
hmm, you are right, must have been a Ctrl+Z malfunction, fixed